weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx

```diff
@@ -203,14 +260,37 @@ const CallPageInnerVertical: FC<{
   return (
     <SimplePageLayoutWithHeader
       headerExtra={
-        <Box>
-          <Button
-            icon="layout-tabs"
-            tooltip={`${showTraceTree ? 'Hide' : 'Show'} trace tree`}
-            variant="ghost"
-            active={showTraceTree ?? false}
-            onClick={onToggleTraceTree}
-          />
+        <Box
```

Comment:
seems like these should only be present if it is peaking AND there are row ids set in the context. I think these arrows will show up on the primary call page right now

Reply:
Yup, fixed.

Reply:
<img width="839" alt="Screenshot 2024-11-04 at 07 58 30" src="https://github.com/user-attachments/assets/18ebdf75-a783-4440-b7b7-762a46ed1a74">
This flex-box puts the tree expansion to the left when full screen (since space-between) will left justify if there is a single element.